crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -370,6 +371,135 @@ impl IndexerStore for PgIndexerStore {
         }
     }
 
+    fn get_all_objects_page(
+        &self,
+        object_id: ObjectID,
+        limit: usize,
+        is_descending: bool,
+    ) -> Result<Vec<ObjectRead>, IndexerError> {
```

Comment:
this will hit the same problem with get_owned_objects imo, I think object pagination should always go to object_history table, cc @patrickkuo